airbyte-integrations/connectors/source-twilio-taskrouter/source_twilio_taskrouter/twilio_taskrouter.yaml

```diff
@@ -0,0 +1,36 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://taskrouter.twilio.com"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['account_sid'] }}"
+      password: "{{ config['auth_token'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  workspaces_stream:
```

Comment:
https://taskrouter.twilio.com/v1/Workspaces/{WorkspaceSid}/Activities
https://www.twilio.com/docs/taskrouter/api/activity#read-multiple-activity-resources

https://taskrouter.twilio.com/v1/Workspaces/{WorkspaceSid}/Events
https://www.twilio.com/docs/taskrouter/api/event#list-all-events

Adding worksapce stream is a good start, is it okay to add more?

Reply:
I'll try to add a few more